frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/AgentOptionsPage/_styles.scss

```diff
@@ -1 +1,16 @@
-.agent-options {}
+.agent-options {
+  font-size: $x-small;
+  a {
+    color: #6a67fe;
+    font-size: 0.875rem;
```

Comment:
please use one of the scss variables from fonts.scss here. same for font-weight

Reply:
💯